lib/engine/game/g_1860.rb

```diff
@@ -175,24 +176,57 @@ def event_fishbourne_to_bank!
         @log << "#{ffc.name} is now available for purchase from the Bank"
       end
 
-      def corp_bankrupt?(corp)
+      def insolvent?(corp)
+        @insolvent_corps.include?(corp)
+      end
+
+      def make_insolvent(corp)
+        return if insolvent?(corp)
+
+        @insolvent_corps << corp
+        @log << "#{corp.name} is now Insolvent"
+      end
+
+      def clear_insolvent(corp)
+        return unless insolvent?(corp)
+
+        @insolvent_corps.delete(corp)
+        @log << "#{corp.name} is no longer Insolvent"
+      end
+
+      def bankrupt?(corp)
         @bankrupt_corps.include?(corp)
       end
 
+      def make_bankrupt(corp)
+        @bankrupt_corps << corp unless bankrupt(corp)
+      end
+
+      def clear_bankrupt(corp)
+        @bankrupt_corps.delete(corp)
+      end
+
+      def status_str(corp)
+        status = nil
```

Comment:
you don't need to declare variables in ruby like this

Reply:
OK